dvc/command/pkg.py

```diff
@@ -54,36 +119,89 @@ def add_parser(subparsers, parent_parser):
-        "'prepare.dvc', 'clean.dvc', etc). "
-        "By default the file has 'mod_' prefix and imported package name "
-        "followed by .dvc",
+    pkg_uninstall_parser.set_defaults(func=CmdPkgUninstall)
+
+    PKG_ADD_HELP = "Add package."
+    pkg_add_parser = pkg_subparsers.add_parser(
+        "add",
+        parents=[parent_config_parser, parent_parser],
+        description=append_doc_link(PKG_ADD_HELP, "pkg-add"),
+        help=PKG_ADD_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
     )
-    pkg_install_parser.set_defaults(func=CmdPkgInstall)
+    pkg_add_parser.add_argument("name", help="Package name.")
+    pkg_add_parser.add_argument("url", help="Package URL.")
+    pkg_add_parser.set_defaults(func=CmdPkgAdd)
+
+    PKG_REMOVE_HELP = "Remove package."
+    pkg_remove_parser = pkg_subparsers.add_parser(
+        "remove",
+        parents=[parent_config_parser, parent_parser],
+        description=append_doc_link(PKG_REMOVE_HELP, "pkg-remove"),
+        help=PKG_REMOVE_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    pkg_remove_parser.add_argument("name", help="Package name.")
+    pkg_remove_parser.set_defaults(func=CmdPkgRemove)
+
+    PKG_MODIFY_HELP = "Modify package."
+    pkg_modify_parser = pkg_subparsers.add_parser(
+        "modify",
+        parents=[parent_config_parser, parent_parser],
+        description=append_doc_link(PKG_MODIFY_HELP, "pkg-remove"),
+        help=PKG_MODIFY_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    pkg_modify_parser.add_argument("name", help="Package name.")
+    pkg_modify_parser.add_argument("option", help="Option.")
+    pkg_modify_parser.add_argument("value", nargs="?", help="Value.")
+    pkg_modify_parser.add_argument(
+        "-u",
+        "--unset",
+        default=False,
+        action="store_true",
+        help="Unset option.",
+    )
+    pkg_modify_parser.set_defaults(func=CmdPkgModify)
+
+    PKG_LIST_HELP = "List packages."
+    pkg_list_parser = pkg_subparsers.add_parser(
+        "list",
+        parents=[parent_config_parser, parent_parser],
+        description=append_doc_link(PKG_LIST_HELP, "pkg-list"),
+        help=PKG_LIST_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    pkg_list_parser.set_defaults(func=CmdPkgList)
+
+    PKG_IMPORT_HELP = "Import data from package."
+    pkg_import_parser = pkg_subparsers.add_parser(
+        "import",
+        parents=[parent_parser],
+        description=append_doc_link(PKG_IMPORT_HELP, "pkg-import"),
+        help=PKG_IMPORT_HELP,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+    )
+    pkg_import_parser.add_argument("pkg_name", help="Package name.")
+    pkg_import_parser.add_argument(
+        "pkg_path", help="Path to data in the package."
+    )
+    pkg_import_parser.add_argument(
+        "out", nargs="?", help="Destination path to put data to."
```

Comment:
It is not clear how I suppose to install all the data files from a package in a single command.

Reply:
You can't do that with `install`. What you are looking for is `dvc pkg import`, which doesn't support directories yet. Will be fixed by https://github.com/iterative/dvc/issues/1861 .

Reply:
I think Dmitry's question was how do `import *`. `pkg_path` is required in the current implementation. It's hard to tell for me how important the functionality of importing everything at once is, but in general, languages do support this. And it can be convenient from the UX perspective. The biggest question here is how should it look like on the DVC-file level (it was a file with multiple outputs in the Dmitry's implementation).